docs/examples.rst

```diff
@@ -1,17 +1,182 @@
+*********
 Examples
-========
-
-.. toctree::
-   :maxdepth: 1
-
-   Launching jobs on Google Cloud <https://github.com/google/flax/tree/main/examples/cloud>
-   ImageNet Classification <https://github.com/google/flax/tree/main/examples/imagenet>
-   Language Modeling (lm1b) <https://github.com/google/flax/tree/main/examples/lm1b>
-   MNIST Classification <https://github.com/google/flax/tree/main/examples/mnist>
-   Part-of-Speech Tagging <https://github.com/google/flax/tree/main/examples/nlp_seq>
-   Predicting Biological Activities of Molecules with Graph Neural Networks <https://github.com/google/flax/tree/main/examples/ogbg_molpcba>
-   Proximal Policy Optimization <https://github.com/google/flax/tree/main/examples/ppo>
-   Seq2Seq: Addition <https://github.com/google/flax/tree/main/examples/seq2seq>
-   SST-2 Classification <https://github.com/google/flax/tree/main/examples/sst2>
-   Basic VAE <https://github.com/google/flax/tree/main/examples/vae>
-   Machine Translation <https://github.com/google/flax/tree/main/examples/wmt>
\ No newline at end of file
+*********
+
+Core examples
+##############
+
+
+Core examples are hosted on the Flax repo in the `examples <https://github.com/google/flax/tree/main/examples>`__ directory.
+
+Each example is designed to be **self-contained and easily forkable**, while
+reproducing relevant results in different areas of machine learning.
+
+As discussed in `#231 <https://github.com/google/flax/issues/231>`__, we decided to go for a standard pattern for all examples
+including the simplest ones (like MNIST). This makes every example a bit more
+verbose, but once you know one example, you know the structure of all of them.
+Having unit tests and integration tests is also very useful when you fork these
+examples.
+
+Some of the examples below have a link "🕹Interactive🕹" that lets you run them
```

Comment:
Let's remove one of the 🕹 ... this looks a bit too agitated :)

(see how I change my mind? ;)

Reply:
Done!